NICE.Identity.Test/UnitTests/Authorisation.WebAPI/Services/UserServiceTests.cs

```diff
@@ -205,6 +206,57 @@ public void Get_users_with_filter_not_found()
             usersFilterNotFound.ShouldNotBe(null);
         }
 
+        private void AddUsersWithAndWithoutRoles(IdentityContext context)
+        {
+	        context.Environments.Add(new DataModels.Environment() { EnvironmentId = 1, Name = "Test" });
+	        context.Services.Add(new DataModels.Service() { ServiceId = 1, Name = "Service" });
+	        context.Websites.Add(new DataModels.Website()
+	        {
+		        WebsiteId = 1,
+		        EnvironmentId = 1,
+		        ServiceId = 1,
+		        Host = "test.nice.org.uk"
+	        });
+	        context.Roles.Add(new DataModels.Role() { RoleId = 1, WebsiteId = 1, Name = "TestRole" });
+	        context.Users.Add(new DataModels.User() { UserId = 1, NameIdentifier = "user with role" });
+	        context.UserRoles.Add(new DataModels.UserRole() { UserRoleId = 1, RoleId = 1, UserId = 1 });
+
+	        context.Users.Add(new DataModels.User() { UserId = 2, NameIdentifier = "user without role" });
+
+            context.SaveChanges();
+        }
+
+        [Fact]
+        public void Get_users_returns_users_without_roles()
```

Comment:
Should that be returns all users rather than returns users without roles as it is returning a user with a role? Might be misunderstanding something though.

Reply:
Also you seem to be testing findusers not getusers but the test name suggests get users

Reply:
nice spot. i've renamed it a bit.